hnn_core/tests/test_gui.py

```diff
@@ -1,8 +1,9 @@
 # Authors: Huzi Cheng <hzcheng15@icloud.com>
-from hnn_core.gui.gui import run_hnn_gui
+from hnn_core.gui.gui import HNNGUI
 
 
 def test_run_gui():
     """Test if main gui function gives proper ipywidget"""
-    gui = run_hnn_gui()
-    assert gui is not None
+    gui = HNNGUI()
+    app_layout = gui.run()
+    assert app_layout is not None
```

Comment:
can you add a couple of tests using the new class so we can see how the new structure helps?

Reply:
check the latest commit